### ISSUE TITLE
scrollEl accepting only a selector makes it difficult to use stickybits with libraries like react

### ISSUE THREAD

**Issue by cmalven** [dollarshaveclub/stickybits#340]
React has a concept of `refs` which can be used to work with javascript libraries that depend on DOM elements being more reliably available.

The `scrollEl` parameter in stickybits currently takes on a selector string, which makes it impossible to point to a specific rendered element in React and be confident that element isn't going to change out from under stickybits.

How would you feel about updating `scrollEl` to accept either a selector _or_ an element to make it easier to use with React?

**Comment by yowainwright** [comment]
I feel indifferent. If you'd like to submit a PR with what you'd like, I will definitely look. 😁 

If you'd like to get insight into how other open source repos use Stickybits you can check this link [out](https://www.npmjs.com/browse/depended/stickybits) (it looks like some of them are React based). 

—Thanks!

**Comment by cmalven** [comment]
@yowainwright I figured that'd probably be the case. I submitted a pretty simple PR that makes all of the changes necessary. I didn't bother to update the release info/changelog yet, but happy to do so if you're at all interested in merging.

I also looked through all of the open source libs you listed and none of them are using the `scrollEl` param, so they wouldn't have run into this. Thanks for sharing those, though.
